clang/test/Analysis/ctu-on-demand-parsing.c

```diff
@@ -24,6 +24,7 @@
 //
 // FIXME: Path handling should work on all platforms.
 // REQUIRES: system-linux
+// UNSUPPORTED: target={{.*}}-zos{{.*}}
```

Comment:
Would this work instead? I think this testcase won't work for all non-linux targets

// REQUIRES: system-linux && target={{.*linux.*}}

Reply:
Unless we can prove that is the case, it is better to stick with the more selective condition.  I don't think many people run lit cross target so it would be difficult to validate.